sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1516,7 +1539,7 @@ def task_worker(self):
                         # D.1.3 Software Configuration and Initialization
                         api.set_datapath_init(host_lanes)
                         # TODO: Use fine grained timeout when the CMIS memory map is available
-                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.CMIS_DEF_EXPIRED)
+                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.get_cmis_expired(api))
```

Comment:
@abohanyang : get_cmis_expired() - is this a temporary function defined to validate this fix - #293?

practically,  get_datapath_init_duration() and  get_datapath_init_duration() and get_datapath_deinit_duration() to be invoked instead. 
Would you be taking care of this as part of this PR? or shall we take care of this in xcvrd.py via #294 ?

Reply:
Yes. I just commit more changes in this PR to call get_datapath_init_duration and get the expiration time.

Reply:
> Yes. I just commit more changes in this PR to call get_datapath_init_duration and get the expiration time.

OK.
@abohanyang : can you please attach UT cases, logs for this case (400G ZR optics FSM working via xcrd.py as orchestrator with these timeouts all the way to interface in 'operational up' state)

Reply:
[log.txt](https://github.com/sonic-net/sonic-platform-daemons/files/9709709/log.txt)